sdk/personalizer/Azure.AI.Personalizer/src/Models/RlObjectConverter.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+using Rl.Net;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Azure.AI.Personalizer
+{
+    /// <summary> The converter between objects for Rl.Net objects and the sdk </summary>
+    internal static class RlObjectConverter
+    {
+        /// <summary>
+        /// Convert PersonalizerRankOptions object to a json context string for Rl.Net
+        /// </summary>
+        public static string ConvertToContextJson(PersonalizerRankOptions personalizerRankOptions)
```

Comment:
DecisionContext contains information about the entire rank request, context and actions. I think you should change the name to ConvertToDecisionContext for clarity

Reply:
This is actually matching the parameter name of the method definition in LiveModel:
public RankingResponse ChooseRank(string eventId, string contextJson, ActionFlags flags);
So I feel this is better.